lnrpc/switchrpc/switch_server.go

```diff
@@ -58,12 +58,21 @@ var (
 			Entity: "offchain",
 			Action: "write",
 		}},
+		"/switchrpc.Switch/TrackOnion": {{
+			Entity: "offchain",
+			Action: "read",
+		}},
 	}
 
 	// DefaultSwitchMacFilename is the default name of the switch macaroon
 	// that we expect to find via a file handle within the main
 	// configuration file in this package.
 	DefaultSwitchMacFilename = "switch.macaroon"
+
+	// ErrAmbigousPaymentState is an error returned when a payment attempt
+	// completes in an ambiguous state: no error and no valid preimage.
+	ErrAmbigousPaymentState = errors.New("payment completed in an " +
```

Comment:
what does it mean then if we end up in this state, can you give more context in the comment how we could end up in this state ?